lua/orgmode/capture/init.lua

```diff
@@ -292,8 +307,13 @@ function Capture.autocomplete_refile(arg_lead)
 end
 
 function Capture:kill()
-  vim.cmd([[autocmd! OrgCapture BufWipeout <buffer>]])
-  vim.api.nvim_win_close(0, true)
+  vim.api.nvim_create_autocmd('BufWipeout', {
```

Comment:
Here you actually need to remove the `BufWipeout` command to avoid it from triggering once the window is closed. I believe you need to save the initial `BufWipeout` into a variable and then delete it via `nvim_del_autocmd()`

Reply:
It does have the `once` flag set to true. With that in mind, is removing it still necessary?

Reply:
Yeah, because when you trigger this `Capture:kill()` function, BufWipeout wasn't triggered even once. `nvim_win_close` would trigger it, and it would cause it to save something that shouldn't be saved.

Reply:
Should I delete it in the same call `Capture:kill()`?

Reply:
Yes, this old command `vim.cmd([[autocmd! OrgCapture BufWipeout <buffer>]])` basically removes the autocmd defined previously.